LCM/scripts/Register.py

```diff
@@ -186,7 +191,7 @@ def usage():
 metaConfig = metaConfig.replace("<CONFIGURATIONMODE>", ConfigurationMode)
 metaConfig = metaConfig.replace("<SERVERURL>", ServerURL)
 metaConfig = metaConfig.replace("<REGKEY>", RegistrationKey)
-
+metaConfig = metaConfig.replace("<CONFIGURATIONNAME>", ConfigurationName)
```

Comment:
Even though the MOF property is specifically titled "ConfigurationNames", DSC only supports one configuration name.  I didn't want to confuse users into thinking they could specify more than one name here :)